index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <header> <h1>Hello Microverse!</h1></header>   
```

Comment:
- [ ] You don't have to make  `h1` tag wrapped by `header`. Please delete `header` tag.